lib/benchmarks/scroll.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Have you considered using https://api.flutter.dev/flutter/flutter_test/WidgetController-class.html to manipulate widgets programmatically (e.g. you can use `drag` to scroll).

Reply:
`scroll.dart` contains a few helper functions for scrolling, such as `scrollUntilVisible`, which controls whether the scroll is animated, and how the target widget is aligned with the viewport. It's not very complicated, so I think we can keep it here.